README.md

```diff
@@ -63,9 +68,30 @@ Packmap is a node package that is used as follows:
 const packmap = require("packmap");
 
 const options = {
+<<<<<<< HEAD
```

Comment:
Is it because of unexpected merge?

Reply:
Oop! Yep. Good catch - i've updated this.

Reply:
Oop! Yep. Good catch - i've updated this.